hspec-core/src/Test/Hspec/Core/Formatters/Pretty/Parser.hs

```diff
@@ -16,12 +16,6 @@ import           Prelude ()
 import           Test.Hspec.Core.Compat hiding (fail)
 import           Test.Hspec.Core.Formatters.Pretty.Parser.Types
 
-#ifndef __GHCJS__
-#if __GLASGOW_HASKELL__ >= 802 && __GLASGOW_HASKELL__ <= 904
```

Comment:
I think we want to keep the check for `<= 904` (either here or in `package.yaml`), so that Hspec keeps building  with  upcoming versions of GHC (changes to the GHC API are the norm, not the exception).

Reply:
This can also be expressed in the .cabal file with upper bound in impl(ghc), assuming that hpack makes it expressible.

Reply:
Yeah sorry, I was so distracted I didn't even read your comment fully. I added appropriate upper bound to `package.yaml`.